src/screens/Overview/components/Stats/utils/removeRevenueMonth/index.test.js

```diff
@@ -1,4 +1,4 @@
-import removeCurrentMonth from '.'
+import removeRevenueMonth from '.'
```

Comment:
Rename this as it doesn't have to do with the current month, but rather removing a month from the revenue object